### DISCUSSION TITLE
No Engine

### DISCUSSION THREAD

**Discussion by Adi-18** [flybywiresim/aircraft#8364] [Help] [answered]
What's is wrong here? It's the first time have this:
![image](https://github.com/flybywiresim/aircraft/assets/29123730/62cc3c9a-e9cd-415d-b6d0-ecb432380968)

![image](https://github.com/flybywiresim/aircraft/assets/29123730/d1177e85-cc5a-472a-ae1a-644ddb6e1d89)

**Comment by Adi-18** [comment]
After restart it works again, but I lost my flight plan...

**Comment by ExampleWasTaken** [comment, marked as answer]
There's nothing wrong. The Xs indicate that the FADECs are unpowered. They are automatically managed so you don't have to worry about it.

**Comment by Maximilian-Reuter** [comment]
> What's is wrong here? It's the first time have this: ![image](https://private-user-images.githubusercontent.com/29123730/293596633-62cc3c9a-e9cd-415d-b6d0-ecb432380968.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNzEwNDgsIm5iZiI6MTcxNTM3MDc0OCwicGF0aCI6Ii8yOTEyMzczMC8yOTM1OTY2MzMtNjJjYzNjOWEtZTljZC00MTVkLWI2ZDAtZWNiNDMyMzgwOTY4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEwVDE5NTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA4ZTQ2NjY4MWJhMDMzNGFjZDAyZGRjMjVkMDcwOWVlM2YxMThmYTZiNDY3MzQ0ODFjODczZjY2MzNmOTgyNDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.unNA6wtUU3Kh_QKXZTEKTVe3cggvzmPiL1b7hq6eGjg)
> 
> ![image](https://private-user-images.githubusercontent.com/29123730/293596648-d1177e85-cc5a-472a-ae1a-644ddb6e1d89.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUzNzEwNDgsIm5iZiI6MTcxNTM3MDc0OCwicGF0aCI6Ii8yOTEyMzczMC8yOTM1OTY2NDgtZDExNzdlODUtY2M1YS00NzJhLWFlMWEtNjQ0ZGRiNmUxZDg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTEwVDE5NTIyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg1NmQ5OGYyNzA3NmIzYWMxZTIzMDhlM2NkYzBhZmNjZTIzYjM0Y2NkYmNiZWY3MTY0NThlZTY0ZjY1NmNhOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vMThhckFtg6VLHI3MNvVa5UUIvZX4beouVNqUqnM54w)

As Example was saying, the FADECs, the computers in the engines simply got unpowered because you waited a while. They automatically power off 5 minutes after startup of the aircraft. That is totally normal. If you turn the Ignition Switch to IGN/Start they will power on again .
In the end there is nothing wrong and you can just continue as normal ;)
